### DISCUSSION TITLE
How can I do this

### DISCUSSION THREAD

**Discussion by newintownreal** [KadeArchive/Kade-Engine#1797] [Q&A]
how can I disable weeks that are locked, and how can I remove weeks

**Comment by Gostar64** [comment]
You need to get the source code and go into StoryMenuState.hx to remove weeks.
But to unlock weeks, I'm pretty sure you have to beat the tutorial.

**Reply by newintownreal** [reply]
i meant to disable week locks

**Reply by Gostar64** [reply]
I think you have to find a thing that says weekUnlocked();
and put true in the parentheses.
If that doesn't work, idk then.

**Reply by newintownreal** [reply]
but the code is public static var weekUnlocked:Array<Bool> = []; and idk where to put true

**Reply by lespylol** [reply]
remove "#if debug" and "#end" on StoryMenuState.hx
